data/template/buf.go-client.gen.yaml

```diff
@@ -40,5 +40,16 @@ plugins:
     opt:
       - paths=source_relative
       - named_go_package=api=github.com/bufbuild/buf/private/gen/proto/api
+      - named_go_package=connect=github.com/bufbuild/buf/private/gen/proto/connect
       - named_go_package=connectclient=github.com/bufbuild/buf/private/gen/proto/connectclient
+    strategy: all
+  - name: go-apiclientconnect
+    out: private/gen/proto/apiclientconnect
+    opt:
+      - paths=source_relative
+      - named_go_package=api=github.com/bufbuild/buf/private/gen/proto/api
       - named_go_package=connect=github.com/bufbuild/buf/private/gen/proto/connect
+      - named_go_package=connectclient=github.com/bufbuild/buf/private/gen/proto/connectclient
```

Comment:
Why does this generator need `connectclient` and `connect` ?

Going to come back to this PR and dive a bit more into the generated code, but figured I'd drop this comment to follow up.